### DISCUSSION TITLE
How can I query for all products that reference another product LID?

### DISCUSSION THREAD

**Discussion by jordanpadams** [NASA-PDS/pds-api#232] [Q&A]
How would I query the API for all products that reference `urn:nasa:pds:context:investigation:mission.orex`?

**Comment by pds-ops** [comment]
Here is an example of this query using curl:

```bash
curl --get 'https://pds.nasa.gov/api/search/1.0/products' \
    --data-urlencode 'limit=200' \
    --data-urlencode 'q=((pds:Internal_Reference.pds:lid_reference eq "urn:nasa:pds:context:investigation:mission.orex") or (pds:Internal_Reference.pds:lid_reference like "urn:nasa:pds:context:investigation:mission.orex::*"))' | json_pp
```

Eventually we will have some sort of dashboard or search interface to support this. But for now, curl (or Python) is your best bet.

**Reply by jordanpadams** [reply]
For any other LID queries, replace `urn:nasa:pds:context:investigation:mission.orex` with the LID of your choice.
